UploadDaemon_Test/SymbolAnalysis/SymbolCollectionResolverTest.cs

```diff
@@ -33,19 +33,44 @@ public void TearDown()
         }
 
         [Test]
-        public void ConsidersSymbolFileIncludes()
+        public void ConsidersSymbolFileIncludesFromSymbolDirectory()
         {
             SymbolCollection collection = resolver.ResolveFromSymbolDirectory(TestSymbolDirectory,
                    new GlobPatternList(new List<string> { "DoesNotExist*" }, new List<string> {}));
 
             Assert.That(collection.IsEmpty, Is.True);
-        }
 
-        [Test]
+			collection = resolver.ResolveFromSymbolDirectory(TestSymbolDirectory,
+				   new GlobPatternList(new List<string> { "Cqse.Teamscale.Profiler.Commons" }, new List<string> { }));
+
+			Assert.That(collection.IsEmpty, Is.False);
+		}
+
+		[Test]
+		public void ConsidersSymbolFileIncludesFromTraceFile()
+		{
+
+			ParsedTraceFile traceFile = new ParsedTraceFile(new string[]
+			{
+			$"Assembly=Cqse.Teamscale.Profiler.Commons:2 Version:1.0.0.0 Path:{TestSymbolFilePath}",
+			}, "cov.txt");
+
+			SymbolCollection collection = resolver.ResolveFromTraceFile(traceFile, "@AssemblyDir",
+				   new GlobPatternList(new List<string> { "DoesNotExist*" }, new List<string> { }));
```

Comment:
I would add blank lines only to start a new group of statements. The assert below belongs to the collection before the blank line.